### ISSUE TITLE
Can't compile the source code under Windows 10.

### ISSUE THREAD

**Issue by omonim2007** [NovaSquirrel/NovaTheSquirrel2#68]
Can't compile the game under Windows 10.

Every time after entering "make ." command in shell I have a message:

"pwd" is not internal or external
command, executable program, or batch file.
make: Nothing to be done for '.'.

I used Chocolate, MinGW and some several other ways to run make command, but the result is always the same.

**Comment by NovaSquirrel** [comment]
On Windows I use [msys2](https://www.msys2.org/) to build it. I'll make a note of that in the readme.

**Comment by omonim2007** [comment]
Much better now.

After some dozens of operations (~1 min) it stopped with this error:

gcc audio/brr/gssbrr.c -o audio/brr/gssbrr
process_begin: CreateProcess(NULL, gcc audio/brr/gssbrr.c -o audio/brr/gssbrr, ...) failed.
make (e=2): The specified file cannot be found.
make: *** [audio/brr/gssbrr] Error 2

**Comment by NovaSquirrel** [comment]
That's an error about not having a C compiler. I went and added it to the repo so people don't need to get a compiler just for one utility.

**Comment by omonim2007** [comment]
It seems I cold beat it ))

I installed a GCC and it corrected some libraries. Now I have compiled the game.

Is it normally we haven't in-game sound? And first screen is corrupted (image attached):

![nova-the-squirrel-2_new000](https://github.com/NovaSquirrel/NovaTheSquirrel2/assets/33754489/279f83c4-982a-47ad-a732-efc6a0ec820d)

**Comment by NovaSquirrel** [comment]
Yeah there's no sound yet. It doesn't look like that on a real SNES or any emulator I've tried so I don't know what's going on there.

**Comment by john-pericos** [comment]
Here are steps that worked for me (as of November 2023):
Install WSL and Ubuntu (Ubuntu 22.04.3 LTS codename: jammy) in Windows 10
Run the following to install the dependencies in Ubutntu terminal
`sudo apt-get install libsdl2-dev
sudo apt-get install libsdl2-image-dev
sudo apt-get install libsdl2-ttf-dev`
Then go to the directory where you have PrincessEdit2 source and run **make**
After that you should be able to find editor built in the root of the PrincessEdit2 directory
Execute the editor like so:
`./editor slopedemo.json`
